nyt-top-20-science-news/src/components/NewsFeed/NewsFeed.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import Row from 'react-bootstrap/Row';
+import StoryCard from '../StoryCard';
+
+/**
+ * The section where news stories will be populated.
+ * @param {Object} param0 - The news container.
+ * @param {Object.<string, string>[]} param0.news - The list of news stories
```

Comment:
is this what flow type definitions looks like? having never encountered this notation before, it's not immediately clear why there are duplicate strings `<string, string>`. I know it's not surprising that someone would prefer the tool they're already familiar with, but my gut reaction is that TypeScript's type syntax is a little more elegant and intuitive.

Reply:
The motivation here is to me a little more granular than `{Object}`. TypeScript probably reads much easier. 

This took me a while to figure out but it's just jsDoc. 
From what I understand, it reads: an array of objects of key type string and value type string. It reads a bit similar to Java. 

Some resources: 
- [Google style guide](https://google.github.io/styleguide/javascriptguide.xml#JsTypes)
- [StackOverflow](https://stackoverflow.com/a/31573441)